src/connect-zwa-2/getting-started/connect-the-device-to-ha-via-proxy-device.md

```diff
@@ -1,8 +1,47 @@
 ---
 zendesk:
   article_id: 29751691427741
-  name: Connecting the device to Home Assistant via a proxy device
+  name: Using a proxy device to connect Home Assistant Connect ZWA-2
   position: 49
   labels: connect-zwa-2
 ---
 
+If you need to place Connect ZWA-2 far away from your Home Assistant system, you can run Z-Wave JS on another system and connect that to Home Assistant.
+
+This allows you to plug Home Assistant Connect ZWA-2 in a location that is closer to your end devices.
+
+## Prerequisites
+
+- A system that can act as proxy and fulfills the following requirements:
```

Comment:
This system needs a free USB-A port (or else the requirement on line 22 will not work)